generators/client/templates/angular/src/main/webapp/app/admin/audits/_audits.service.ts

```diff
@@ -36,6 +39,6 @@ export class AuditsService  {
             search: params
         };
 
-        return this.http.get('<% if (authenticationType === 'uaa') { %><%= uaaBaseName.toLowerCase() %>/<% } %>management/audits', options);
+        return this.http.get(<% if (authenticationType === 'uaa') { %>'<%= uaaBaseName.toLowerCase() %>/<% } else { %>SERVER_API_URL + '<% } %>management/audits', options);
```

Comment:
This is the only endpoint that should have the UAA condition.  All of the other files you changed should just use SERVER_API_URL

Reply:
oh yes you are right totally forgot about that when suggesting something else above

Reply:
@ruddell thanks for the feedback. I'm waiting for #6824 and @deepu105 's decision before doing an update.